app/controllers/triannon/auth_controller.rb

```diff
@@ -0,0 +1,353 @@
+require_dependency "triannon/application_controller"
+
+module Triannon
+  # Adapted from http://image-auth.iiif.io/api/image/2.1/authentication.html
+  class AuthController < ApplicationController
+    include RdfResponseFormats
+
+    AUTH_EXPIRY  =   60 # seconds
+    TOKEN_EXPIRY = 3600 # seconds
```

Comment:
@azaroth42 - please consider whether these auth requirements for timeouts are fixed values that can be hard-coded or variable values that could and should be config parameters?  As I've interpreted it, they are fixed values that are not negotiable and @ndushay raises a good question about whether that should be so.